test/externalTests/colony.sh

```diff
@@ -34,7 +34,7 @@ function colony_test
     FORCE_ABIv2=false
     CONFIG="truffle.js"
 
-    truffle_setup https://github.com/erak/colonyNetwork.git develop_060
+    truffle_setup https://github.com/aarlt/colonyNetwork.git natspec_on_state_variables
```

Comment:
We recently moved the external tests to https://github.com/solidity-external-tests. Can you make these changes there?